### ISSUE TITLE
Simplify miniInfo.*

### ISSUE THREAD

**Issue by matanlurey** [swlegion/tts#238]
Specifically:

1. We should be able to infer `collider` from the `unitInfo.baseSize`, right?
2. We should be able to infer / merge `name` with the upgrade card or unit card.
3. It appears in almost every or every case `red` and `blue` are the same.
4. We should just have `scale` _always_ be `{1, 1, 1}`.

**Comment by nashjaee** [comment]
> Specifically:
> 
> 1. We should be able to infer `collider` from the `unitInfo.baseSize`, right?

Definitely. We'll have to refactor [this table](https://github.com/swlegion/tts/blob/76994246dcc03a65cda8f1c098cc39b4dcd637ee/mod/src/StarWarsLegion.lua#L1562) to not be a local variable, but that's a good idea.

> 2. We should be able to infer / merge `name` with the upgrade card or unit card.

This seems reasonable. We'll have to concatenate " Leader" for unit leader minis, but I think that's the only edge case.

> 3. It appears in almost every or every case `red` and `blue` are the same.

I'm almost positive the only exception is the ATST. I think it's ok to merge these.

> 4. We should just have `scale` _always_ be `{1, 1, 1}`.

Agreed

**Comment by matanlurey** [comment]
Update:

- Getting rid of `collider` and  `name` might require some work; it's definitely possible but convoluted.
- I removed `blue` in every case where it was duplicate with `red` and updated the code accordingly.
- Removing `.scale` was super easy.

**Comment by matanlurey** [comment]
Unassigning as I'm not working on the remaining items right now.
